sample.mvn.parent/pom.xml

```diff
@@ -14,10 +16,43 @@
 	</parent>
 
 	<properties>
```

Comment:
For the properties in general, can you suggest if we can upgrade any of the version numbers? 
I already upgrade the bnd-maven-plugin version from 3.0.0 to 5.0.1, but the other ones are more obscure to me in terms of what the runtime might need/expect to run without issues

Reply:
I can't provide any guideline on that I'd suggest to do things step by step

Reply:
I will do that